utils/regex/regex_file_generator.cr

```diff
@@ -0,0 +1,17 @@
+macro generate_source(data)
+<<-EOT
+# THIS FILE IS AUTOMATICALLY GENERATED, DO NOT EDIT !
+
+module Emoji
+  REGEX = /{{data.id}}/
```

Comment:
Cool! I am testing this out. 

Can we rename that to `EMOJI_REGEX` to save backward compatibility? That was a public constant though.